packages/components/switch/src/switch.vue

```diff
@@ -120,6 +112,39 @@ useDeprecated(
   },
   computed(() => !!vm.vnode.props?.value)
 )
+useDeprecated(
+  {
+    from: '"active-color"',
+    replacement: 'CSS var `--el-switch-on-color`',
+    scope: COMPONENT_NAME,
+    version: '2.3.0',
+    ref: 'https://element-plus.org/en-US/component/switch.html#attributes',
+    type: 'Attribute',
+  },
+  computed(() => !!vm.vnode.props?.activeColor)
+)
+useDeprecated(
+  {
+    from: '"inactive-color"',
+    replacement: 'CSS var `--el-switch-off-color`',
+    scope: COMPONENT_NAME,
+    version: '2.3.0',
+    ref: 'https://element-plus.org/en-US/component/switch.html#attributes',
+    type: 'Attribute',
+  },
+  computed(() => !!vm.vnode.props?.inactiveColor)
+)
+useDeprecated(
+  {
+    from: '"border-color"',
+    replacement: 'CSS var `--el-switch-border-color`',
+    scope: COMPONENT_NAME,
+    version: '2.3.0',
+    ref: 'https://element-plus.org/en-US/component/switch.html#attributes',
+    type: 'Attribute',
+  },
+  computed(() => !!vm.vnode.props?.borderColor)
+)
```

Comment:
There's some repetitive code here, perhaps it can be extracted. 🤔

Reply:
What do you think of the following?
```js
type DeprecationTypes = Parameters<typeof useDeprecated>
const useBatchDeprecated = (
  params: Array<
    Pick<DeprecationTypes[0], 'from' | 'replacement'> & {
      condition: DeprecationTypes[1]
    }
  >
) => {
  params.forEach((param) => {
    useDeprecated(
      {
        from: param.from,
        replacement: param.replacement,
        scope: COMPONENT_NAME,
        version: '2.3.0',
        ref: 'https://element-plus.org/en-US/component/switch.html#attributes',
        type: 'Attribute',
      },
      param.condition
    )
  })
}
useBatchDeprecated([
  {
    from: '"value"',
    replacement: '"model-value" or "v-model"',
    condition: computed(() => !!vm.vnode.props?.value),
  },
  {
    from: '"active-color"',
    replacement: 'CSS var `--el-switch-on-color`',
    condition: computed(() => !!vm.vnode.props?.activeColor),
  },
  {
    from: '"inactive-color"',
    replacement: 'CSS var `--el-switch-off-color`',
    condition: computed(() => !!vm.vnode.props?.inactiveColor),
  },
  {
    from: '"border-color"',
    replacement: 'CSS var `--el-switch-border-color`',
    condition: computed(() => !!vm.vnode.props?.borderColor),
  },
])
```

Reply:
Maybe this is more concise: 
```ts
const useBatchDeprecated = (list: string[][]) => {
  list.forEach((param) => {
    useDeprecated(
      {
        from: param[0],
        replacement: param[1],
        scope: COMPONENT_NAME,
        version: '2.3.0',
        ref: 'https://element-plus.org/en-US/component/switch.html#attributes',
        type: 'Attribute',
      },
      computed(() => !!vm.vnode.props?[param[2]])
    )
  })
}
useBatchDeprecated([
 ['value', '"model-value" or "v-model"', 'value'],
 ...
])
```

Reply:
updated